letsencrypt/plugins/manual.py

```diff
@@ -130,7 +130,7 @@ def _perform_single(self, achall):
         if self.conf("test-mode"):
             logger.debug("Test mode. Executing the manual command: %s", command)
             # sh shipped with OS X does't support echo -n
-            if sys.platform == "darwin":
+            if sys.platform == "darwin":   # pragma: no cover
                 executable = "/bin/bash"
             else:
                 executable = None
```

Comment:
Doesn't that just feel like cheating the code coverage detector?
Better would probably be to copy a test somewhere and mock `sys.platform`, I suppose.